src/core.c

```diff
@@ -2673,19 +2687,19 @@ int GetMouseWheelMove(void)
 // Returns touch position X for touch point 0 (relative to screen size)
 int GetTouchX(void)
 {
-#if defined(PLATFORM_ANDROID) || defined(PLATFORM_WEB)
+#if defined(PLATFORM_ANDROID) || defined(PLATFORM_WEB) || defined(PLATFORM_UWP)
     return (int)CORE.Input.Touch.position[0].x;
-#else   // PLATFORM_DESKTOP, PLATFORM_RPI
+#else   // PLATFORM_DESKTOP, PLATFORM_RPI, PLATFORM_UWP
```

Comment:
Just a typo I think, if PLATFORM_UWP is considered on previous #if, it shouldn't be here in the comment

Reply:
Oh, sorry. I thought that was talking about what the if contained. I shall remove these